scripts/manage-versions.ts

```diff
@@ -77,8 +81,8 @@ function saveDataFiles(dictionary, versions) {
     currentVersion: versions[0],
   };
   fs.writeFileSync(dataFilename, JSON.stringify(content));
-  const treeData = generateTreeData(content.dictionary);
-  fse.writeJSONSync(dataFileTreeName, treeData);
+  //const treeData = generateTreeData(content.dictionary);
+  //fse.writeJSONSync(dataFileTreeName, treeData);
```

Comment:
worth keeping around or should we just remove?

Reply:
worth keeping for now, i think tree will be worked on pretty soon